lib/modifiers.js

```diff
@@ -67,7 +80,17 @@ function nestedRedact (store, o, path, ns, censor, isCensorFct, censorFctTakesPa
 }
 
 function has (obj, prop) {
-  return Object.prototype.hasOwnProperty.call(obj, prop)
+  return obj !== undefined && obj !== null
+    ? Object.prototype.hasOwnProperty.call(obj, prop)
+    : false
+}
+
+function isIn (obj, prop) {
+  try {
+    return prop in obj
+  } catch (e) {
+    return false
+  }
```

Comment:
is this not slower than inlining `typeof b === 'object'  && a in b` wherever it's needed?

Reply:
Might be - is fixed